modules/custom/az_event/config/install/core.entity_form_display.node.az_event.default.yml

```
@@ -42,11 +42,12 @@ third_party_settings:
       format_type: details_sidebar
       format_settings:
         classes: ''
+        show_empty_fields: false
         id: ''
         open: true
-        description: 'For use in feeds and content listings.'
+        description: 'Summary fields determine how this event appears in feeds, listings and other displays throughout the site. Utilize these options to tailor the presentation and effectiveness of your content in various contexts.'
```

I think the first sentence is sufficient for all the content types. This description could also link to a quickstart.arizona.edu page on content summaries once that is ready.

```suggestion
        description: 'Summary fields determine how this event appears in feeds, listings and other displays throughout the site.'
```